<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>666</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Chat becomes available after Auth eID login</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageActive">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,6,2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="usersFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="headerPanel" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="SearchWidget" name="searchWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ChatUserListTreeView" name="treeViewUsers">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideMiddle</enum>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="indentation">
             <number>10</number>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>false</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <property name="expandsOnDoubleClick">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>User: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelUserName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>offline</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="chatFrame">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="darkFrame" stdset="0">
          <bool>true</bool>
         </property>
         <property name="headerPanel" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelActiveChat">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>CHAT #</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="headerLabel" stdset="0">
               <bool>false</bool>
              </property>
              <property name="headerPanel" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidgetMessages">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QFrame" name="page_6">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QWidget" name="chatTopSeparatorWidget" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>1</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>1</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="messagesHeaderFrame">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>25</height>
                   </size>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeType">
                      <enum>QSizePolicy::Fixed</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>8</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="timeLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>101</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Time</string>
                     </property>
                     <property name="minimumSizeLinux" stdset="0">
                      <size>
                       <width>110</width>
                       <height>0</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="iconLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>20</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="userLabel">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>90</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>User</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="messageLabel">
                     <property name="minimumSize">
                      <size>
                       <width>50</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Message</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="ChatMessagesTextEdit" name="textEditMessages">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="focusPolicy">
                   <enum>Qt::NoFocus</enum>
                  </property>
                  <property name="autoFillBackground">
                   <bool>true</bool>
                  </property>
                  <property name="autoFormatting">
                   <set>QTextEdit::AutoAll</set>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="acceptRichText">
                   <bool>false</bool>
                  </property>
                  <property name="openExternalLinks">
                   <bool>true</bool>
                  </property>
                  <property name="openLinks">
                   <bool>false</bool>
                  </property>
                  <property name="darkFrame" stdset="0">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frameContactActions">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QPushButton" name="pushButton_AcceptSend">
                     <property name="text">
                      <string>ACCEPT/SEND</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushButton_RejectCancel">
                     <property name="text">
                      <string>REJECT/CANCEL</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="inputContainer">
                  <property name="darkFrame" stdset="0">
                   <bool>true</bool>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="BSChatInput" name="input_textEdit">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>56</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>84</height>
                      </size>
                     </property>
                     <property name="readOnly">
                      <bool>false</bool>
                     </property>
                     <property name="acceptRichText">
                      <bool>true</bool>
                     </property>
                     <property name="placeholderText">
                      <string>TYPE MESSAGE AND SEND</string>
                     </property>
                     <property name="openExternalLinks">
                      <bool>true</bool>
                     </property>
                     <property name="openLinks">
                      <bool>false</bool>
                     </property>
                     <property name="darkFrame" stdset="0">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QFrame" name="page_7">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="TreeViewWithEnterKey" name="treeViewOTCRequests">
                  <property name="focusPolicy">
                   <enum>Qt::WheelFocus</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="rootIsDecorated">
                   <bool>true</bool>
                  </property>
                  <property name="itemsExpandable">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="roomsFrame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QStackedWidget" name="stackedWidgetOTC">
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="OTCLoginRequiredShield" name="widgetLoginRequiredShield"/>
            <widget class="OTCGeneralRoomShield" name="widgetGeneralShield"/>
            <widget class="CreateOTCRequestWidget" name="widgetCreateOTCRequest"/>
            <widget class="PullOwnOTCRequestWidget" name="widgetPullOwnOTCRequest"/>
            <widget class="CreateOTCResponseWidget" name="widgetCreateOTCResponse"/>
            <widget class="OTCNegotiationRequestWidget" name="widgetNegotiateRequest"/>
            <widget class="OTCNegotiationResponseWidget" name="widgetNegotiateResponse"/>
            <widget class="OTCParticipantShield" name="widgetParticipantShield"/>
            <widget class="OTCContactShield" name="widgetContactShield"/>
            <widget class="OTCContactNetStatusShield" name="widgetContactNetStatusShield"/>
            <widget class="OTCSupportRoomShield" name="widgetSupportShield"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChatMessagesTextEdit</class>
   <extends>QTextBrowser</extends>
   <header>ChatUI/ChatMessagesTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TreeViewWithEnterKey</class>
   <extends>QTreeView</extends>
   <header>TreeViewWithEnterKey.h</header>
  </customwidget>
  <customwidget>
   <class>CreateOTCRequestWidget</class>
   <extends>QWidget</extends>
   <header>CreateOTCRequestWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CreateOTCResponseWidget</class>
   <extends>QWidget</extends>
   <header>CreateOTCResponseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCNegotiationResponseWidget</class>
   <extends>QWidget</extends>
   <header>OTCNegotiationResponseWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCNegotiationRequestWidget</class>
   <extends>QWidget</extends>
   <header>OTCNegotiationRequestWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCLoginRequiredShield</class>
   <extends>QWidget</extends>
   <header>OTCLoginRequiredShield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCGeneralRoomShield</class>
   <extends>QWidget</extends>
   <header>OTCGeneralRoomShield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PullOwnOTCRequestWidget</class>
   <extends>QWidget</extends>
   <header>PullOwnOTCRequestWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchWidget</class>
   <extends>QWidget</extends>
   <header location="global">ChatUI/SearchWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCParticipantShield</class>
   <extends>QWidget</extends>
   <header>OTCParticipantShield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCContactShield</class>
   <extends>QWidget</extends>
   <header>OTCContactShield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCContactNetStatusShield</class>
   <extends>QWidget</extends>
   <header>OTCContactNetStatusShield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OTCSupportRoomShield</class>
   <extends>QWidget</extends>
   <header>OTCSupportRoomShield.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatUserListTreeView</class>
   <extends>QTreeView</extends>
   <header>ChatUI/ChatUserListTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>BSChatInput</class>
   <extends>QTextBrowser</extends>
   <header>ChatUI/BSChatInput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
